app/views/docs/functions.phtml

```diff
@@ -384,6 +384,35 @@ func main(req: RequestValue, res: RequestResponse) throws -> RequestResponse {
     <p>If you're running Appwrite locally and trying to connect to your local Appwrite instance from a cloud function, you'll have to use your local IP address in place of localhost. This is because, within the container localhost does not refer to the host machine rather the container itself. </p>
 </div>
 
+<h2><a href="/docs/functions#ignoreFiles" id="ignoreFiles">Ignore Files</a></h2>
+
+<p>Library folders such as <b>node_modules</b> or <b>vendor</b> should not be included in the tarball, although you will want them locally so you have syntax highlighting in your IDE. Similarly, you should <b>not</b> include files containing secrets in your deployment. You can use the Appwite CLI's <b>file ignoring</b> feature to exclude specific files from a deployment.</p>
```

Comment:
```suggestion
<p>Library folders such as <b>node_modules</b> or <b>vendor</b> should not be included in your tarball since these dependencies will be installed during your functions build process. Similarly, you should <b>not</b> include files containing secrets in your deployment. You can use the Appwite CLI's <b>file ignoring</b> feature to exclude specific files from a deployment.</p>
```

Reply:
Your chanes makes sense, but they remove part where we explain why people want them locally (IDE support). Can we remove that?

Reply:
Yes, it's okay to remove it. It's not very important in the context of this doc right ?